modules/ami/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "ami_name" {
+  description = "Name of the AMI to use"
+  type = string
```

Comment:
i think this is not necessary in the new version (the current version we are working)

this works, is just a comment

Reply:
I added to so we could use any other OS version. Current was Ubuntu 18.04, I'm using 20.04 but I didn't want to hardcode into the module.